gapis/api/vulkan/api/pipeline.api

```diff
@@ -664,12 +675,23 @@ cmd void vkDestroyPipeline(
 // Shader modules //
 ////////////////////
 
+@internal class UsedDescriptor {
```

Comment:
I think it would be clearer if we name this type `DescriptorInfo`, and the variables of this type named `UsedDescriptor`.